work/sources_list.md

```diff
@@ -0,0 +1,107 @@
+# List of Sources to Work With
+
+Below you will find the list of useful sources I have found in the task and elsewhere. Some of these will require little to no work to bring over to the portal, others will need more effort.
+
+These are categorized by the portal where the information is contained and NOT by category (although I did try to sort them roughly). Some sources might be agglomerated into one article, others could potentially be used in multiple places.
+
+Once you consider the source extinguished, please mark it with the check `[x]`. Ideally, by the end of the project all these sources should be used (or at the very least consulted).
```

Comment:
We'll remove `docs` folders from both repos when their content will be moved here.